lagom/interfaces.py

```diff
@@ -198,4 +198,5 @@ def reflection_cache_overview(self) -> Dict[str, str]:
     SpecialDepDefinition[T],  # From the definitions module
     SpecialDepDefinition[NoReturn],  # Some types don't resolve
     T,  # Just an instance of the type - A singleton
+    List[Callable],  # A collection of functions - maps to FunctionCollection[F]
```

Comment:
Maybe I don't need this "magic" at all :thinking:

Reply:
yeah I don't like this magic. I  think I'll remove this and just provide FunctionCollection as a helper